src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala

```diff
@@ -45,7 +45,9 @@ class CloudStackVaultController(vaultService: VaultService,
   private val logger = LoggerFactory.getLogger(this.getClass)
   private val accountEntityName = "accounts"
   private val vmEntityName = "vms"
-  private val vaultApiPath = s"${vaultService.endpoint}${RequestPath.vaultRoot}"
+  private val vaultApiPathsString = vaultService.endpoints.map { endpoint =>
+    s"$endpoint${RequestPath.vaultRoot}"
```

Comment:
use https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html for a path concatenation

Reply:
remove unused method "createMissingAccountTokenTag()"

Reply:
ok, nice advice ;)